### PR TITLE
Fix for issue #1456

### PR DESCRIPTION
**Pull request by rogerpereira123**
Fixing #1456

---
This change is [Reviewable](https://reviewable.io/reviews/ratpack/ratpack/1457)

### CODE REVIEW

File: ratpack-retrofit2/src/main/java/ratpack/retrofit/RatpackRetrofitCallException.java [hunk 1]
```diff
@@ -57,7 +58,11 @@ public RatpackRetrofitCallException(Call<?> call, String error) {
    * @since 1.6.0
    */
   public RatpackRetrofitCallException(Request request, Response<?> response) {
-    super(request.url().toString() + ": " + Exceptions.uncheck(() -> response.errorBody().string()));
+    super(request.url().toString() + ": " + Exceptions.uncheck(() -> {
+      Charset charset = response.errorBody().contentType() == null ? null : response.errorBody().contentType().charset();
+      if (charset == null) charset = Charset.forName("UTF-8");
```
Comment (johnrengelman) [inline ratpack-retrofit2/src/main/java/ratpack/retrofit/RatpackRetrofitCallException.java]: This should use `{}` after the `if`

Comment (johnrengelman) [inline ratpack-retrofit2/src/main/java/ratpack/retrofit/RatpackRetrofitCallException.java]: You will need to use `{}` here to wrap the body of the `if` statement in order to comply with the style check in this project.

Comment (rogerpereira123) [reply ratpack-retrofit2/src/main/java/ratpack/retrofit/RatpackRetrofitCallException.java]: Done!

**Comment by johnrengelman** [comment]
Merged in https://github.com/ratpack/ratpack/pull/1458

**Pull request closed without merging**